dataactbroker/handlers/accountHandler.py

```diff
@@ -137,6 +144,117 @@ def login(self,session):
         return self.response
 
 
+    def max_login(self,session):
+        """
+
+        Logs a user in if their password matches
+
+        arguments:
+
+        session  -- (Session) object from flask
+
+        return the reponse object
+
+        """
+        try:
+            safeDictionary = RequestDictionary(self.request)
```

Comment:
Sorry for not being clear. What I'm questioning is...why does the broker need a special class like this to process an incoming request, rather than using whatever request-process means are included in the Flask framework?